Make clean not cleaning up Wear module

#6699 introduced a specific module targeting Android wear devices. This works in a similar fashion as the MapboxGLAndroidSDKTestApp for phone/tablets. When switching from release to master branch and trying to execute `make android` currently shows:

```java
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':MapboxGLAndroidSDKWearTestApp:transformNative_libsWithStripDebugSymbolForDebug'.
> java.io.IOException: Failed to delete /Users/nurbot/mapbox-gl-native/platform/android/MapboxGLAndroidSDKWearTestApp/build/intermediates/transforms/stripDebugSymbol/debug/folders/2000/1f/main/lib/x86

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

```

It seems that our make clean command is not correctly cleaning up:

```bash
rm -rf ./build \
	        ./platform/android/MapboxGLAndroidSDK/build \
	        ./platform/android/MapboxGLAndroidSDKTestApp/build \
	        ./platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/gen \
	        ./platform/android/MapboxGLAndroidSDK/src/main/jniLibs \
	        ./platform/android/MapboxGLAndroidSDKTestApp/src/main/jniLibs \
	        ./platform/android/MapboxGLAndroidSDK/src/main/assets

```

--- a/Makefile
+++ b/Makefile
@@ -547,6 +547,7 @@ clean:
 	-rm -rf ./build \
 	        ./platform/android/MapboxGLAndroidSDK/build \
 	        ./platform/android/MapboxGLAndroidSDKTestApp/build \
+	        ./platform/android/MapboxGLAndroidSDKWearTestApp/build \
 	        ./platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/gen \
 	        ./platform/android/MapboxGLAndroidSDK/src/main/jniLibs \
 	        ./platform/android/MapboxGLAndroidSDKTestApp/src/main/jniLibs \

